### PR TITLE
Update notes for rate limiting configuration settings

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Comment by jasonblais** [comment]
@coreyhulen and @esethna please help with review?

**Comment by jasonblais** [comment]
@coreyhulen I see, I can add this to the rate limiting documentation instead.

Just to confirm, are the following recommended settings for rate limiting regardless of what environment you have:
 - **Maximum Queries per Second**: 10
 - **Maximum Burst Size**: 100
 - **Memory Store Size**: Set to the number of users in the system
 - **Vary rate limit by remote address**: Set to ``true`` if you're using a proxy
 - **Vary rate limit by HTTP header**: Set to ``true`` if you're using a proxy

**Comment by coreyhulen** [comment]
I'm not sure  *Vary rate limit by remote address *and *Vary rate limit by
HTTP header *are needed because they depend on the type of proxy.  Might be
better to give a recommend setting for nginx.

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -1930,15 +1930,17 @@ Maximum number of requests allowed beyond the per second query limit.
...
+Typically set to the number of users in the system.
 
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | This feature's ``config.json`` setting is ``"MemoryStoreSize": 10000`` with whole number input.                                                                      |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
 Vary rate limit by remote address
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-**True**: Rate limit API access by IP address.
+**True**: Rate limit API access by IP address. Recommended to set to ``true`` if you're using a proxy.
```
Comment (coreyhulen) [inline source/administration/config-settings.rst]: The recommendation should be set to true irrespective of proxy.  If you have a proxy then you have more work todo setting the other fields.

Comment (coreyhulen) [inline source/administration/config-settings.rst]: wait...nvm.

**Comment by lfbrock** [comment]
+1

**Pull request merged**